hephaistus/main-theme/distribution-system/chord/playground/zozond/chord-impl/src/main/kotlin/Hasher.kt

```diff
@@ -0,0 +1,12 @@
+import java.security.MessageDigest
+
+class Hasher {
+
+    companion object {
+        private val digest = MessageDigest.getInstance("SHA-1")
+
+        fun getHashCode(key: String): String{
+            return digest.digest(key.toByteArray()).toString()
+        }
+    }
+}
```

Comment:
```suggestion
private val digest = MessageDigest.getInstance("SHA-1")

fun getHashCode(key: String): String {
    return digest.digest(key.toByteArray()).toString()
}
```
companion `object` create redundant object. The above code corresponds to `static` in java.

If you want to use `Hasher.getHashCode()`, what about it?

 ```suggestion
private val digest = MessageDigest.getInstance("SHA-1")

class Hasher {
        fun getHashCode(key: String): String {
            return digest.digest(key.toByteArray()).toString()
        }
}
```